opm/polymer/fullyimplicit/PolymerPropsAd.cpp

```diff
@@ -192,6 +192,34 @@ namespace Opm {
 
 
 
+    ADB PolymerPropsAd::effectiveInvPolymerVisc(const ADB& c, const double* visc) const
+    {
+        const int nc = c.size();
+        V inv_mu_p_eff(nc);
+        V dinv_mu_p_eff(nc);
+        for (int i = 0; i < nc; ++i) {
+            double im = 0;
+            double dim = 0;
+            // TODO: the usage of visc can be likely wrong, while more investigation will be requried.
```

Comment:
In the comment in the header files, `visc     Array of 2 viscosity value.  `, while from usage it looks like `visc` should be the viscosity of all the water cells. And also from the `PolymerProperties.cpp`, it looks like only `visc[0]` is used. 
I think if we do not change the interface of the functions, here we should use `&(visc[i])`.